client/src/pages/LandingPage.jsx

```diff
@@ -67,11 +70,25 @@ const LandingPage = () => {
 
   const navigate = useNavigate()
 
-  const usersName = 'resumetest';
+  useEffect(() => {
+    const fetchUserSkills = async () => {
+      try {
+        const response = await axios.get(`http://localhost:4000/api/users?userName=${auth.user}`);
```

Comment:
user userID here you shouldnt be exposing the username

Reply:
Undid this change and adjusted to use the  skills get route in users.js